corehq/util/sentry.py

```diff
@@ -137,11 +137,11 @@ def _rate_limit_exc(exc_info):
     except Exception:
         logger.exception("Error while subtyping rate limited error")
 
-    datadog_counter('commcare.sentry.errors.rate_limited', tags=[
-        'service:{}'.format(rate_limit_key)
-    ])
+    metrics_counter('commcare.sentry.errors.rate_limited', tags={
+        'service': rate_limit_key
+    })
     if is_pg_cancelled_query_exception(exc_value):
-        datadog_counter('hq_custom.postgres.standby_query_canellations')
+        metrics_counter('commcare.postgres.standby_query_canellations')
```

Comment:
renamed to use the `commcare` prefix. This isn't in use on any dashboard at present